tools/flaky-test-retryer/README.md

```diff
@@ -97,6 +97,14 @@ automatic retry.
 
 if all retries are expended without a success.
 
+## Updating
+
+1. Run `make push_versioned` in /images/flaky-test-retryer/
+2. Note the image tag, in the format `v[date]-[hash]`.
+3. Replace the current image tag in [retryer_service.yaml](https://github.com/knative/test-infra/blob/master/tools/flaky-test-retryer/gke_deployment/retryer_service.yaml#L40)
+with the tag generated by make, excluding `-dirty` suffix if it exists.
```

Comment:
You should not drop `-dirty`, as it's really part of the flag. Actually, you shouldn't encourage anyone of using a "dirty" image, as it's untrackable.

Reply:
fixed, didn't understand what -dirty meant at first. Should be good now.